### PR TITLE
Feedback Feature included

### PR DESCRIPTION
**Pull request by nemishzalavadiya**

### CODE REVIEW

**Comment by nemishzalavadiya** [comment]
First check the login issues then merge this branch

**Comment by nemishzalavadiya** [comment]
i think because of previous code change there is a problem in login structure

**Comment by nemishzalavadiya** [comment]
now you can start working on admin UI. All major features are done now UI and testing remains.

**Comment by nemishzalavadiya** [comment]
All changes done, Now you can able to merge it

File: OnlineParkingSystem/User/views.py [hunk 1]
```diff
@@ -47,18 +47,19 @@ def Login(request):
...
         email = form.data['email']
         password = form.data['password']
-        user_data=User_detail.objects.get(email=email,password=password,role=request.POST.get('role'))
+        user_data=User_detail.objects.filter(email=email,password=password,role=request.POST.get('role')).first()
         if(user_data):
             request.session['uid']=user_data.userid
             request.session['email']=email
             request.session['role']=request.POST.get('role')
+            print("session started ####@#@@@@ ",request.session['email'])
             return render(request,'index.html',{'login':'True','role':request.POST.get('role')})
```
Comment (shubhamvekariya) [inline OnlineParkingSystem/User/views.py]: Remove this !!

Comment (nemishzalavadiya) [reply OnlineParkingSystem/User/views.py]: done

**Pull request merged**